great_expectations/data_context/data_context.py

```diff
@@ -120,6 +122,10 @@
 from great_expectations.render.renderer.site_builder import SiteBuilder
 from great_expectations.rule_based_profiler import RuleBasedProfiler
 from great_expectations.rule_based_profiler.config import RuleBasedProfilerConfig
+from great_expectations.rule_based_profiler.config.base import (
```

Comment:
@cdkini Per #4111 this will change the location (which is partly why we want to merge that PR first).  Thanks!

Reply:
that's fine - I'll have to perform some merge conflicts but happy to do so